sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmos/CosmosHealthIndicator.java

```diff
@@ -21,13 +20,14 @@
 public class CosmosHealthIndicator extends AbstractHealthIndicator {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CosmosHealthIndicator.class);
+
     @Value("${azure.cosmos.database}")
     private String dbName;
+
     @Value("${azure.cosmos.uri}")
     private String uri;
 
-    @Autowired
```

Comment:
Does the @Autowired annotation can be removed here?

Reply:
Yes, the field is initialized from the constructor.